warp/yul/ToCairoVisitor.py

```diff
@@ -380,10 +382,12 @@ def _make_external_function(
             params += "calldata_size, calldata_len, calldata : felt*"
             init_exec_env = (
                 f"let (returndata_ptr : felt*) = alloc()\n"
-                f"local exec_env : ExecutionEnvironment ="
-                f"ExecutionEnvironment(calldata_size=calldata_size,"
+                f"let (local exec_env : ExecutionEnvironment*) = alloc()\n"
```

Comment:
I think this line can be removed

Reply:
Removed
=

Reply:
Probably will require regenerating files